.github/workflows/security-scan.yml

```diff
@@ -0,0 +1,65 @@
+name: Security Scan
+
+on:
+  push:
+    branches:
+      - main
+      - release/**
+  pull_request:
+    branches:
+      - main
+      - release/**
```

Comment:
We don't PR _from_ `main` and `release/**`, so I don't think we want this filter.

Reply:
I think this should be for PR from a branch to `main` or `release/**`. The addition for release is to trigger the scan on backports too if I am not mistaken. I can remove it if it's not needed?

Reply:
You're right: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpull_requestpull_request_targetbranchesbranches-ignore

> When using the pull_request and pull_request_target events, you can configure a workflow to run only for pull requests that target specific branches.